tsconfig.json

```diff
@@ -15,6 +15,6 @@
     "types": ["vite/client"],
     "allowJs": true
   },
-  "include": ["src/**/*.ts", "src/**/*.d.ts", "src/**/*.vue", "vite.config.ts"],
+  "include": ["src/**/*.ts", "src/**/*.d.ts", "src/**/*.vue", "vite.config.ts", "node_modules/vuestic-ui"],
```

Comment:
Why do we need this? Just curious.

Reply:
We ignore types from node_modules. Because of that, global types are not applied. You're not able to see types for "VaButton" component.